### PR TITLE
Release1.3 autofix

### PR DESCRIPTION
**Pull request by KanchanaAradhya**
Added 5 new rules with its sql script
Added autofix code and its properties into the sql script
Moved all the properties to config

### CODE REVIEW

File: installer/resources/pacbot_app/files/DB.sql [hunk 1]
```diff
@@ -1520,6 +1541,76 @@ INSERT IGNORE INTO pac_config_key_metadata (`cfkey`,`description`) VALUES ('spri
 INSERT IGNORE INTO pac_config_key_metadata (`cfkey`,`description`) VALUES ('spring.mail.properties.mail.smtp.starttls.enable','Description PlaceHolder');
 INSERT IGNORE INTO pac_config_key_metadata (`cfkey`,`description`) VALUES ('spring.mail.test-connection','Description PlaceHolder');
 
+
+INSERT INTO `pac_config_key_metadata` (`cfkey`, `description`) VALUES('pacman.es.host','Description PlaceHolder');
```
Comment (sajeer-nooh) [inline installer/resources/pacbot_app/files/DB.sql]: Please use **INSERT IGNORE INTO**

File: installer/resources/pacbot_app/files/DB.sql [hunk 2]
```diff
@@ -1520,6 +1541,76 @@ INSERT IGNORE INTO pac_config_key_metadata (`cfkey`,`description`) VALUES ('spri
...
+INSERT INTO `pac_config_key_metadata` (`cfkey`, `description`) VALUES('pacman.es.port','Description PlaceHolder');
+INSERT INTO `pac_config_key_metadata` (`cfkey`, `description`) VALUES('esLoggingLevel','Description PlaceHolder');
+insert into `pac_config_key_metadata` (`cfkey`, `description`) values('heimdall-host','Description PlaceHolder');
```
Comment (sajeer-nooh) [inline installer/resources/pacbot_app/files/DB.sql]: To be consistent please use **INSERT IGNORE INTO** (caps)

File: installer/resources/pacbot_app/files/DB.sql [hunk 3]
```diff
@@ -1520,6 +1541,76 @@ INSERT IGNORE INTO pac_config_key_metadata (`cfkey`,`description`) VALUES ('spri
...
+insert into `pac_config_key_metadata` (`cfkey`, `description`) values('pacman.autofix.waittime.PacMan_EC2WithPublicIPAccess_version-1_Ec2WithPublicAccess_ec2','Description PlaceHolder');
+insert into `pac_config_key_metadata` (`cfkey`, `description`) values('pacman.auto.fix.max.email.notifications.PacMan_EC2WithPublicIPAccess_version-1_Ec2WithPublicAccess_ec2','Description PlaceHolder');
+insert into `pac_config_key_metadata` (`cfkey`, `description`) values('pacman.auto.fix.mail.template.columns.PacMan_EC2WithPublicIPAccess_version-1_Ec2WithPublicAccess_ec2','Description PlaceHolder');
+insert into `pac_config_key_metadata` (`cfkey`, `description`) values('pacman.auto.fix.common.email.notifications.PacMan_EC2WithPublicIPAccess_version-1_Ec2WithPublicAccess_ec2','Description PlaceHolder');
+insert into `pac_config_key_metadata` (`cfkey`, `description`) values('pacman.login.user.name','Description PlaceHolder');
+insert into `pac_config_key_metadata` (`cfkey`, `description`) values('pacman.login.password','Description PlaceHolder');
+
+
```
Comment (sajeer-nooh) [inline installer/resources/pacbot_app/files/DB.sql]: Please remove unwanted blank lines

File: installer/resources/pacbot_app/files/DB.sql (hunk 1 above)
Comment (KanchanaAradhya) [reply installer/resources/pacbot_app/files/DB.sql]: Done!!

File: installer/resources/pacbot_app/files/DB.sql (hunk 2 above)
Comment (KanchanaAradhya) [reply installer/resources/pacbot_app/files/DB.sql]: Done!!

File: installer/resources/pacbot_app/files/DB.sql (hunk 3 above)
Comment (KanchanaAradhya) [reply installer/resources/pacbot_app/files/DB.sql]: Done

**Pull request merged**